src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.DevOps.tests.ps1

```diff
@@ -83,6 +83,7 @@ InModuleScope Arcus.Scripting.DevOps {
 
                 # Assert
                 $getResponse = Invoke-WebRequest -Uri $requestUri -Method Get -Headers $headers
+                Write-Host "Response: $($getResponse.Content)"
```

Comment:
Why do we need this in the test? Was this maybe added to debug something?

Reply:
Ah, indeed, debug stuff that should be removed. Thx for spotting.

Reply:
Removed in https://github.com/arcus-azure/arcus.scripting/pull/432/commits/0b9f3d9096cecf747b0ea18f5103b50425df638c